parm/stage/stage.yaml.j2

```diff
@@ -0,0 +1,195 @@
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_RESTART_PREV | relpath(ROTDIR) }}/{{ m_prefix }}.{{ ftype }}", "{{ COMOUT_ATMOS_RESTART_PREV }}"]
+        {% endfor %}
+        {% for ftype in ["ca_data", "fv_core.res", "fv_srf_wnd.res", "fv_tracer.res", "phy_data", "sfc_data"] %}
+            {% for ntile in range(1, ntiles + 1) %}
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_RESTART_PREV | relpath(ROTDIR) }}/{{ m_prefix }}.{{ ftype }}.tile{{ ntile }}.nc", "{{ COMOUT_ATMOS_RESTART_PREV }}"]
+            {% endfor %} # ntile
+        {% endfor %} # ftype
+        {% for ntile in range(1, ntiles + 1) %}
+        {% if path_exists(ICSDIR ~ "/" ~ COMOUT_ATMOS_RESTART_PREV | relpath(ROTDIR) ~ "/" ~ p_prefix ~ ".sfcanl_data.tile" ~ ntile ~ ".nc") %}
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_RESTART_PREV | relpath(ROTDIR) }}/{{ p_prefix }}.sfcanl_data.tile{{ ntile }}.nc", "{{ COMOUT_ATMOS_RESTART_PREV }}"]
+        {% endif %} # path_exists
+        {% endfor %} # ntile
+{% else %} # cold start
+{% set COMOUT_ATMOS_INPUT = COM_ATMOS_INPUT_TMPL | replace_tmpl(current_dict) %}
+atmosphere_cold:
+    mkdir:
+        - "{{ COMOUT_ATMOS_INPUT }}"
+    copy:
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_INPUT | relpath(ROTDIR) }}/gfs_ctrl.nc", "{{ COMOUT_ATMOS_INPUT }}"]
+        {% for ftype in ["gfs_data", "sfc_data"] %}
+            {% for ntile in range(1, ntiles + 1) %}
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_INPUT | relpath(ROTDIR) }}/{{ ftype }}.tile{{ ntile }}.nc", "{{ COMOUT_ATMOS_INPUT }}"]
+            {% endfor %} # ntile
+        {% endfor %} # ftype
+{% endif %}
+
+{% if REPLAY_ICS == True %}
+{% set COMOUT_ATMOS_ANALYSIS = COM_ATMOS_ANALYSIS_TMPL | replace_tmpl(current_dict) %}
+atmosphere_perturbation:
+    mkdir:
+        - "{{ COMOUT_ATMOS_ANALYSIS }}"
+    copy:
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ m_prefix }}.fv3_perturbation.nc", "{{ COMOUT_ATMOS_ANALYSIS }}/{{ RUN }}.t{{ current_cycle_HH }}z.atminc.nc"]
+{% endif %}
+
+{% if DO_NEST %}
+atmosphere_nest:
+  {% set ntile = 7 %}
+  {% if EXP_WARM_START == True %}
+    {% set COMOUT_ATMOS_RESTART_PREV = COM_ATMOS_RESTART_TMPL | replace_tmpl(previous_dict) %}
+    mkdir:
+        - "{{ COMOUT_ATMOS_RESTART_PREV }}"
+    copy:
+        {% for ftype in ["ca_data", "fv_core.res", "fv_srf_wnd.res", "fv_tracer.res", "phy_data", "sfc_data"] %}
+        - ["{{ ICSDIR }}/{{ COMOUT_ATMOS_RESTART_PREV | relpath(ROTDIR) }}/{{ m_prefix }}.{{ ftype }}.tile{{ ntile }}.nc", "{{ COMOUT_ATMOS_RESTART_PREV }}/{{ m_prefix }}.{{ ftype }}.nest0{{ ntile-5 }}.tile{{ ntile }}.nc"]
+        {% endfor %}
+  {% else %} # cold start
+    {% set COMOUT_ATMOS_INPUT = COM_ATMOS_INPUT_TMPL | replace_tmpl(current_dict) %}
+    mkdir:
+        - "{{ COMOUT_ATMOS_INPUT }}"
+    copy:
+        {% for ftype in ["gfs_data", "sfc_data"] %}
+        - ["{{ COMOUT_ATMOS_INPUT }}/{{ ftype }}.tile{{ ntile }}.nc", "{{ COMOUT_ATMOS_INPUT }}/{{ ftype }}.nest0{{ ntile-5 }}.tile{{ ntile }}.nc"]
```

Comment:
Just checking, should the nest number really be tied to `ntile-5` or should it just be `nest02`?

Reply:
Good question. So the current functionality in the staging job script is to set it based on tile-5 in the warm-start block:
```
if (( tt > 6 )) ; then
            tgt="${COM_ATMOS_RESTART_PREV}/${DTG_PREFIX}.${ftype}.nest0$((tt-5)).tile${tt}.nc"
```
https://github.com/NOAA-EMC/global-workflow/blob/develop/scripts/exglobal_stage_ic.sh#L52
...and when cold-start, it hardcodes the resulting "2":
```
if (( ntiles > 6 )); then
        ${NLN} "${COM_ATMOS_INPUT}/${ftype}.tile7.nc" "${COM_ATMOS_INPUT}/${ftype}.nest02.tile7.nc"
```
https://github.com/NOAA-EMC/global-workflow/blob/develop/scripts/exglobal_stage_ic.sh#L82

With the nest, `ntiles=7` so both ways result in the same thing right now = `nest02`.

I chose to go with `nest0{{ ntile-5 }}` to avoid hardcoding the "2" in the filename and to allow for higher `tile` numbers if nests did that. Let me double check with the folks who added the nest functionality though and make sure they didn't need that hardcoded version.

Reply:
@guoqing-noaa Please see my comment above. Does it matter if the filename for the nest IC is built using `nest0{{ ntile-5 }}` or hardcoded with a 2 (`nest02`). Is there a reason to set it differently in different circumstances? Let me know, thanks!